app/views/content/is-teaching-right-for-me/maths/_article.html.erb

```diff
@@ -51,18 +53,14 @@
     name: "Dimitra, maths teacher",
     large: true
   ) %>
-</section>
-    <p><a href="https://www.thenational.academy/teachers/programmes/maths-secondary-ks3-l/units">Explore what teaching a maths lesson would be like</a>.</p>
-    </div>
-</section>
 <section class="col col-845 inset">
```

Comment:
this quote looks quite squashed now
![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/4065bd06-3dba-4b9b-976c-6b08e6958ed1)


depending on your template conversations, I wonder if it is worth getting rid of the heading/image and just having this as a standard quote, like Dimitra's (although perhaps separated out a bit)? but up to you and @jenhadfield-dfe